website/docs/plugins/provider.html.md

```diff
@@ -8,6 +8,13 @@ description: |-
 
 # Provider Plugins
 
+~> **Advanced topic!** Plugin development is a highly advanced
+topic in Terraform, and is not required knowledge for day-to-day usage.
+If you don't plan on writing any plugins, we recommend not reading
```

Comment:
s/we recommend not reading this section of the documentation/this section of the documentation is not necessary to read.